spec/commands/get_review_recommendations_spec.rb

```diff
@@ -102,7 +106,8 @@ def perform_with_predefined_args
 
   context 'requests older than 1 month have been made' do
     let(:other_users_ids) { [other_user1.id, other_user2.id, other_user3.id, other_user4.id] }
-    let!(:pull_request_1) { create(:pull_request_relation, github_user_id: github_user_id, target_user_id: other_user4.id, gh_updated_at: Time.now - 2.months) }
+    let!(:pull_request_1) { create(:pull_request_relation, github_user_id: other_user4.id,
+      target_user_id: github_user_id, gh_updated_at: Time.current - 2.months) }
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Expression at 110, 79 should be on its own line.